ogindia/demographics.py

```diff
@@ -41,21 +40,20 @@ def get_fert(totpers, min_yr, max_yr, graph=False):
             of life
 
     '''
-    # Get current population data (2013) for weighting
+    # Get current population data (2011) for weighting
     cur_path = os.path.split(os.path.abspath(__file__))[0]
+    # Get current population data (2011) for weighting
     pop_file = utils.read_file(cur_path,
-                               "data/demographic/pop_data.csv")
-    pop_data = pd.read_csv(pop_file, sep=',', thousands=',')
-    pop_data_samp = pop_data[(pop_data['Age'] >= min_yr - 1) &
-                             (pop_data['Age'] <= max_yr - 1)]
-    curr_pop = np.array(pop_data_samp['2013'], dtype='f')
+                               "data/demographic/india_pop_data.csv")
```

Comment:
What is going on here?

Reply:
Some more changes are required in demographics to give it a more realistic picture.